### PR TITLE
Create selected-media.html

### PR DESCRIPTION
**Pull request by teolemon**
Selected media teaser for the French press page. Equivalents should be created for countries where we have significant media coverage.

### CODE REVIEW

File: lang/fr/texts/selected-media.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	</div>
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/global/global-rfi.jpeg" alt="Une base de données de produits alimentaires" style="height:145px">
+		<h3>2019 - C'est En France - La révolution du scan</h3>
+		<p>Nous avons présenté Open Food Facts sur France 24 dans le cadre d’ <a href="https://youtu.be/GMBQGG2FjP8?t=440">un reportage sur la révolution alimentaire que nous avons lancée en France</a> ( <a href="https://youtu.be/yDZKs_ibJns?t=441">version française</a> ). 
+		</p>
+	</div>
+</div>
+
+<a class="button" href="/presskit_fr_FR_selection">Toutes nos apparitions selectionnées</a>
```
Comment (hangy) [inline lang/fr/texts/selected-media.html]: That URL doesn't sound like a good idea to me. We shouldn't pollute the root URI with that many different styles, and [Cool URIs don't change](https://www.w3.org/Provider/Style/URI). Wouldn't it be better to host it somewhere else (outside of the actual web app)?

File: lang/fr/texts/selected-media.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="row text-center">
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/lemonde.png" alt="" style="height:145px">
```
Comment (hangy) [inline lang/fr/texts/selected-media.html]: `alt` text should not be empty

File: lang/fr/texts/selected-media.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		<h3>2013 - Le Monde</h3>
+		<p> 
+			<a href="http://www.lemonde.fr/sante/article/2013/04/15/alimentation-face-aux-doutes-les-internautes-s-organisent_3159792_1651302.html">Alimentation: face aux doutes, les internautes s'organisent.</a> Interview croisée d'Open Food Facts et Notéo dans Le Monde.
+		</p>
+	</div>
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/media-france2-envoyespecial.jpeg" alt="" style="height:145px">
```
Comment (hangy) [inline lang/fr/texts/selected-media.html]: `alt` text should not be empty

File: lang/fr/texts/selected-media.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<div class="row text-center">
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/lemonde.png" alt="" style="height:145px">
+		<h3>2013 - Le Monde</h3>
+		<p> 
+			<a href="http://www.lemonde.fr/sante/article/2013/04/15/alimentation-face-aux-doutes-les-internautes-s-organisent_3159792_1651302.html">Alimentation: face aux doutes, les internautes s'organisent.</a> Interview croisée d'Open Food Facts et Notéo dans Le Monde.
```
Comment (hangy) [inline lang/fr/texts/selected-media.html]: Might want to link to their HTTPS endpoint instead.

File: lang/fr/texts/selected-media.html (hunk 2 above)
Comment (teolemon) [inline lang/fr/texts/selected-media.html]:
```suggestion
		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/lemonde.png" alt="Le Monde" style="height:145px">
```

File: lang/fr/texts/selected-media.html (hunk 4 above)
Comment (teolemon) [inline lang/fr/texts/selected-media.html]:
```suggestion
			<a href="https://www.lemonde.fr/sante/article/2013/04/15/alimentation-face-aux-doutes-les-internautes-s-organisent_3159792_1651302.html">Alimentation: face aux doutes, les internautes s'organisent.</a> Interview croisée d'Open Food Facts et Notéo dans Le Monde.
```

File: lang/fr/texts/selected-media.html (hunk 3 above)
Comment (teolemon) [inline lang/fr/texts/selected-media.html]:
```suggestion
		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/media/france/media-france2-envoyespecial.jpeg" alt="Open Food Facts sur Envoyé Spécial" style="height:145px">
```

File: lang/fr/texts/selected-media.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	</div>
+	<div class="medium-4 columns">
+		<img src="https://world.openfoodfacts.org/files/presskit/PressKit/global/global-rfi.jpeg" alt="Une base de données de produits alimentaires" style="height:145px">
+		<h3>2019 - C'est En France - La révolution du scan</h3>
+		<p>Nous avons présenté Open Food Facts sur France 24 dans le cadre d’ <a href="https://youtu.be/GMBQGG2FjP8?t=440">un reportage sur la révolution alimentaire que nous avons lancée en France</a> ( <a href="https://youtu.be/yDZKs_ibJns?t=441">version française</a> ). 
+		</p>
+	</div>
+</div>
+
+<a class="button" href="/presskit_fr_FR_selection">Toutes nos apparitions selectionnées</a>
```
Comment (teolemon) [inline lang/fr/texts/selected-media.html]:
```suggestion
<a class="button" href="/selection-presse">Toutes nos apparitions sélectionnées</a>
```

**Comment by teolemon** [comment]
fixed all concerns

**Pull request merged**